services/compute/virtualmachine/virtualmachine.go

```diff
@@ -922,3 +929,26 @@ func (c *client) getZoneConfiguration(zoneConfiguration *wssdcommon.ZoneConfigur
 		StrictPlacement: &strinctPlacement,
 	}
 }
+
+func (c *client) getWssdPlacementGroupReference(s *compute.PlacementGroupReference) (*wssdcloudcompute.PlacementGroupReference, error) {
+	if s == nil {
+		return nil, nil
+	}
+
+	placementGroup := &wssdcloudcompute.PlacementGroupReference{
+		Name:      *s.Name,
+		GroupName: *s.GroupName,
+	}
+	return placementGroup, nil
+}
+
+func (c *client) getPlacementGroupReference(placementGroup *wssdcloudcompute.PlacementGroupReference) *compute.PlacementGroupReference {
+	if placementGroup == nil {
+		return nil
+	}
+
+	return &compute.PlacementGroupReference{
+		Name:      &placementGroup.Name,
```

Comment:
I think this will assign address of a pointer that is pointing to placementGroup.Name. rather you can directly do `Name:      placementGroup.Name,`

Reply:
same for below

Reply:
Discussed offline